debug.log

```diff
@@ -0,0 +1 @@
+[1220/160211.746:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
May I ask what this is for?

Reply:
I honestly don't know what this is, I didn't even realize I pushed this. Looks like some kind of error though maybe @BraidenCutforth can clear this one up.

Reply:
Yeah it is an error file. Delete it in your branch and push another commit. We don't want it in our repo.